docs/SPIDER_TYPE.md

```diff
@@ -0,0 +1,23 @@
+## Types of spiders
+
+The vast majority of spiders are collecting first party data about a specific brand/chain.
+We sometimes call these "brand spiders", these should be well categorised and branded by us, and have original data from
+the brand.
+We store them in [locations/spiders](../locations/spiders). and  they contain `spider:lineage=S_ATP_BRANDS` in the outputted
+`dataset_attributes`.
+
+We also collect government Open Data.
+These should also contain license details in `dataset_attributes`.
```

Comment:
alltheplaces.xyz states the downloadable weekly run result is CC-0 so setting a license in `dataset_attributes` sends mixed messages. In jurisdictions that ATP users seem to be within, a database of facts is not copyrightable. But if there are some creative fields in source data such as `company motto`, then perhaps this shouldn't be extracted at all so as to not pollute the weekly download on alltheplaces.xyz?